odl/util/phantom.py

```diff
@@ -372,9 +391,9 @@ def derenzo_sources(space):
 
 
 def shepp_logan(space, modified=True):
-    """ Create a Shepp-Logan phantom.
+    """Create a Shepp-Logan phantom.
 
-    The shepp-logan phantom
+    The standard Shepp-Logan phantom in 2 or 3 dimensions.
```

Comment:
standard is missleading since we default to modified. Perhaps we should change that instead?